Fedora/hpx.spec

```diff
@@ -215,34 +286,45 @@ done
 %files openmpi
 %doc README.rst
 %license LICENSE_1_0.txt
-%{_libdir}/openmpi*/lib/lib*.so.*
+%{_libdir}/openmpi*/lib/lib*.so*
 %{_libdir}/openmpi*/lib/%{name}
+%{_libdir}/openmpi*/lib/lib*.a
 
 %files openmpi-examples
 %doc README.rst
 %license LICENSE_1_0.txt
 %{_libdir}/openmpi*/bin/*
 
+%files openmpi-devel
+%{_includedir}/%{name}
+%{_libdir}/openmpi*/lib/pkgconfig/*.pc
+%{_libdir}/openmpi*/lib/cmake/HPX
+%{_libdir}/openmpi*/lib/bazel
 
 %files mpich
 %doc README.rst
 %license LICENSE_1_0.txt
-%{_libdir}/mpich*/lib/lib*.so.*
+%{_libdir}/mpich*/lib/lib*.so*
```

Comment:
The `.so` files (symlinks) have to go in the devel packages.

Reply:
Moved